### DISCUSSION TITLE
Nextjs

### DISCUSSION THREAD

**Discussion by kamalkech** [oscartbeaumont/rspc#53] [General]
follow react doc and get this error:
Error: Cannot find package 'react-dom' imported from /demo-rspc/node_modules/@tanstack/react-query/build/lib/reactBatchedUpdates.mjs

**Comment by oscartbeaumont** [comment]
Do you have `react-dom` in your `package.json`?

Maybe try running `pnpm i react-dom`.

**Reply by kamalkech** [reply]
yes i have react-dom, when i read doc i found
npm i @rspc/client
pnpm i @rspc/react 

why need to use different registry package ?!!

**Reply by oscartbeaumont** [reply]
Me showing both package managers is a mistake. You can use either `npm`, `yarn` or `pnpm` for all commands so feel free to replace the start of the command with whatever package manager you use.
